src/test/java/net/bhardy/bizzo/billing/PolicyBuilderTest.java

```diff
@@ -33,233 +33,197 @@
  *
  */
 public class PolicyBuilderTest {
+    private final LocalDate friday4th = LocalDate.of(2017, 8, 4);
+    private final LocalDate saturday5th = LocalDate.of(2017, 8, 5);
+    private final LocalDate sunday6th = LocalDate.of(2017, 8, 6);
+    private final LocalDate monday7th = LocalDate.of(2017, 8, 7);
+    private final LocalDate tuesday8th = LocalDate.of(2017, 8, 8);
+
     @Test
     public void cycleType_monthlySimple() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)